source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -213,6 +218,24 @@ class StopAreaSerializer(serpy.DictSerializer):
     name = NestedPropertyField(attr='properties.geocoding.name', display_none=True)
     administrative_regions = AdministrativeRegionsSerializer()
     timezone = NestedPropertyField(attr='properties.geocoding.timezone')
+    commercial_modes = jsonschema.MethodField(display_none=True)
```

Comment:
you have a `NestedPropertyField` can you use that one?

Reply:
In the response bragi is of type dict but not pbfield